projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -127,6 +132,8 @@ public function __construct() {
 		$this->is_rtl          = isset( $this->user_data['text_direction'] ) && 'rtl' === $this->user_data['text_direction'];
 		$this->user_locale     = isset( $this->user_data['user_locale'] ) ? $this->user_data['user_locale'] : '';
 		$this->site_woa        = ( new Host() )->is_woa_site();
+		// update true to check user meta. true -> get_user_meta( $this->user_data['ID'], 'wpcom-global-nav-enabled', true ); or get_user_option( 'wpcom-global-nav-enabled', $this->user_id )
+		$this->use_dev_nav = $this->site_woa && true;
```

Comment:
I am having issues with checking user meta/options here in testing currently.

I have added this meta/option to my user ID and verified via my sandbox. They both return as '1'.  However, in my WoA dev environment this does not seem to be updated as this logs empty or false.

(note - in the end we would just launch this to all sites satisfying `$this->site_woa`, but to gate this for prior internal testing we want to check a meta/option)

Reply:
Asking about this in slack. It seems that not all user meta is sync'd and that may be the issue.  p1707248035041979-slack-CDLH4C1UZ

Reply:
This is handled via D137411-code